src/client/ClientDataResolver.js

```diff
@@ -84,7 +84,7 @@ class ClientDataResolver {
    * Data that resolves to give a GuildMember object. This can be:
    * * A GuildMember object
    * * A User object
-   * @typedef {Guild} GuildMemberResolvable
+   * @typedef {GuildMember} GuildMemberResolvable
```

Comment:
If the point is to resolve a GuildMember then why give a GuildMember if you already have it...